__tests__/__src__/environment/TestEnvironment.ts

```diff
@@ -45,11 +47,69 @@ export class TestEnvironment extends BaseTestEnvironment {
             nodePath.resolve(__dirname, "../../__resources__/application_instances"),
             process.env.PATH
         ].join(separator);
-
+        result.resources = {
+            localFiles: [],
+            files: [],
+            jobs: [],
+            jobData: [],
+            datasets: [],
+            session: result.resources?.session || null
+        };
         // Return the test environment including working directory that the tests should be using
         return result;
     }
 
+    /**
+     * Clean up your test environment.
+     * Deletes any temporary profiles that have been created
+     * @params {ITestEnvironment} testEnvironment - the test environment returned by createTestEnv
+     *
+     * @returns {Promise<void>} - promise fulfilled when cleanup is complete
+     * @throws errors if profiles fail to delete
+     * @memberof TestEnvironment
+     */
+    public static async cleanUp(testEnvironment: ITestEnvironment<ITestPropertiesSchema>) {
+        if (testEnvironment.tempProfiles != null) {
+            await TempTestProfiles.deleteProfiles(testEnvironment);
+        }
+        if (testEnvironment.pluginInstalled) {
+            const pluginDir = testEnvironment.workingDir + "/plugins";
+            require("rimraf").sync(pluginDir);
+        }
```

Comment:
Instead of reimplementing the cleanup code from the base test environment, we could invoke the super class here:
```suggestion
        await super.cleanup(testEnvironment);
```